site/content/deployment/enmasse.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "OpenShift"
+weight = 475
++++
+
+## Build Hono
+
+~~~sh
+~/hono$ mvn clean install -Pbuild-docker-image
+~~~
+
+## Deploy EnMasse
+
+Dowload the release, unpack and run
+
+~~~sh
+./deploy-openshift.sh -n hono
+~~~
+
+## Deploy Hono
+
+~~~sh
+~hono/example/target/deploy/openshift$ chmod +x *
+~hono/example/target/deploy/openshift$ ./enmasse_deploy.sh
+~~~
+
+## Extract Certificate
+
+~~~sh
+~hono/example$ oc extract secret/external-certs-messaging --to=target/config/hono-demo-certs-jar/ -n hono
+~~~
+
+## Start Consumer
+
+~~~sh
+~hono/example$ mvn spring-boot:run -Drun.arguments=--hono.client.host=messaging-hono.$(minishift ip).nip.io,--hono.client.port=443,--hono.client.trustStorePath=target/config/hono-demo-certs-jar/server-cert.pem,--hono.client.username=consumer@HONO,--hono.client.password=verysecret
+~~~
+
+## Register Device
+
+~~~sh
+~hono/example$ curl -X POST -i -H 'Content-Type: application/json' -d '{"device-id": "4711"}' http://hono-service-device-registry-http-hono.$(minishift ip).nip.io/registration/DEFAULT_TENANT
+~~~
+
+## Send Telemetry
+
+~~~sh
+~hono/example$ curl -X POST -i -u sensor1@DEFAULT_TENANT:hono-secret -H 'Content-Type: application/json' --data-binary '{"temp": 5}' http://hono-adapter-rest-vertx-hono.$(minishift ip).nip.io/telemetry
```

Comment:
node port?

Reply:
Ditto as above.

Reply:
The routes work like this for standard http requests, there's no need for port

Reply:
Ditto as above ;)